CONTRIBUTING.md

```diff
@@ -173,6 +173,47 @@ The `legacy_foo` arg was removed
 :::
 ```
 
+## Style and idioms
+
+For the most part, we just accept whatever the code formatters do, so there
+isn't much style to enforce.
+
+Some miscellanous style, idioms, and conventions we have are:
+
+### Markdown/Sphinx Style
+
+* Use colons (e.g. `:::`) for prose sections of text.
```

Comment:
Do you want to include a note on the prose type? Eg `{topic}`, `{note}`, `{warning}`, etc.

Reply:
Done